src/main/kotlin/org/rust/ide/presentation/Utils.kt

```diff
@@ -76,6 +90,7 @@ private fun presentableName(psi: RsElement): String? {
                 append(typeParameterBounds(psi))
             }
         }
+        is RsLetDecl -> (psi.pat as? RsPatIdentImpl)?.patBinding?.name
```

Comment:
And a more interesting problem: your implementation doesn't handle let declarations with complex patterns. E.g. `let (a,b) = (1,2)` will be ignored in the variables view. But I think it should not. At least, Python Console supports tuple patterns in declarations. Probably you should collect all the bindings like `val bindings = letDecl.pat.descendantsOfType<RsPatBinding>()` and then handle each binding separately.

Reply:
Yes, fixed. Now all variables (`RsPatIdent`) are collected in `RsStructureViewModel::getVariablesDeclarations`